lib/file.coffee

```diff
@@ -43,6 +40,12 @@ class File
   onDidStatusChange: (callback) ->
     @emitter.on('did-status-change', callback)
 
+  loadRealPath: ->
```

Comment:
No, I'm totally cool using promises in this package.

I just didn't see why it was needed in this specific case.